public/.htaccess

```diff
@@ -0,0 +1,93 @@
+</Files>
+
+# Route errors to static pages automatically generated by SilverStripe
+#ErrorDocument 404 /assets/error-404.html
+ErrorDocument 404 /public/errors/error-404.html
+ErrorDocument 500 /assets/error-500.html
+
+<IfModule mod_rewrite.c>
+
+    # Turn off index.php handling requests to the homepage fixes issue in apache >=2.4
+    <IfModule mod_dir.c>
+        DirectoryIndex disabled
+        DirectorySlash On
+    </IfModule>
+
+    SetEnv HTTP_MOD_REWRITE On
+    RewriteEngine On
+
+    # Enable HTTP Basic authentication workaround for PHP running in CGI mode
+    RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization}]
+
+    # Deny access to potentially sensitive files and folders
+    RewriteRule ^vendor(/|$) - [F,L,NC]
+    RewriteRule ^\.env - [F,L,NC]
+    RewriteRule silverstripe-cache(/|$) - [F,L,NC]
+    RewriteRule composer\.(json|lock) - [F,L,NC]
+    RewriteRule (error|silverstripe|debug)\.log - [F,L,NC]
+
+    ### =============== Legacy redirects ===============
+
+    # Packages no longer exist
+    RewriteCond %{REQUEST_URI} ^(.*)en/.*/package-.*\.html [NC]
+    RewriteRule ^(.*)$ %1 [L,R=302]
+
+    # 2.x no longer exists
+    RewriteCond %{REQUEST_URI} ^(.*)en/2[.] [NC]
+    RewriteRule ^(.*)$ %1 [L,R=302]
+
+    # Redirect non-languaged minor versions to major version, e.g. /3.1/DataObject.html to /3/DataObject.html
+    RewriteCond %{REQUEST_URI} ^(.*)(\d)[.]\d/(.*)$ [NC]
+    RewriteRule ^(\d)\.\d/(.*)$ %1%2/%3 [L,R=302]
+
+    # Redirect language minor version to major version, e.g. /en/3.1/File.html to /3/File.html
+    RewriteCond %{REQUEST_URI} ^(.*)en/([\d]+)[.][\w.]+/(.*)$ [NC]
+    RewriteRule ^(.*)$ %1%2/%3 [L,R=302]
+
+    # Class redirect. E.g.
+    # /4/class-SilverStripe.AssetAdmin.Forms.FileSearchFormFactory.html
+    # /4/SilverStripe/AssetAdmin/Forms/FileSearchFormFactory.html
+    RewriteCond %{REQUEST_URI} ^(.*)([\w.]+)/class-([\w]+)\.html [NC]
+    RewriteRule ^(.*)$ %1%2/%3.html [L,R=302]
+    RewriteCond %{REQUEST_URI} ^(.*)([\w.]+)/class-([\w]+)\.([\w]+)\.html [NC]
+    RewriteRule ^(.*)$ %1%2/%3/%4.html [L,R=302]
+    RewriteCond %{REQUEST_URI} ^(.*)([\w.]+)/class-([\w]+)\.([\w]+)\.([\w]+)\.html [NC]
+    RewriteRule ^(.*)$ %1%2/%3/%4/%5.html [L,R=302]
+    RewriteCond %{REQUEST_URI} ^(.*)([\w.]+)/class-([\w]+)\.([\w]+)\.([\w]+)\.([\w]+)\.html [NC]
+    RewriteRule ^(.*)$ %1%2/%3/%4/%5/%6.html [L,R=302]
+    RewriteCond %{REQUEST_URI} ^(.*)([\w.]+)/class-([\w]+)\.([\w]+)\.([\w]+)\.([\w]+)\.([\w]+)\.html [NC]
+    RewriteRule ^(.*)$ %1%2/%3/%4/%5/%6/%7.html [L,R=302]
+    RewriteCond %{REQUEST_URI} ^(.*)([\w.]+)/class-([\w]+)\.([\w]+)\.([\w]+)\.([\w]+)\.([\w]+)\.([\w]+)\.html [NC]
+    RewriteRule ^(.*)$ %1%2/%3/%4/%5/%6/%7/%8.html [L,R=302]
+
+    ### ============= End Legacy redirects =============
+
+    # Lookup script (used to be a SS app, now just a simple script)
+    # E.g. http://api.silverstripe.org.loc/search/lookup/?q=SilverStripe\ORM\HasManyList&version=4&module=framework
```

Comment:
The `&module=` example was removed from the readme - if that functionality no longer works, remove that part from this example as well.

Reply:
Removed